### PR TITLE
SAK-41459 Fixed translation loading delay

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-41437

Fixed site title and creator name under Edge

### CODE REVIEW

**Comment by adrianfish** [comment]
https://jira.sakaiproject.org/browse/SAK-41437

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ export class SakaiRubricCriteria extends SakaiElement {
     criterion.ratings = sortedIds.map(id => criterion.ratings.find(r => r.id == id));
 
     var baseUrl = `${window.location.protocol}//${window.location.host}/rubrics-service/rest/ratings/`;
-    const urlList = sortedIds.reverse().reduce((a, id) => { return `${baseUrl}${id}\n${a}` }, '');
+    const urlList = sortedIds.reverse().reduce((a, id) => { return `${baseUrl}org.sakaiproject.rubrics:rubrics-service:war:20-SNAPSHOT\n${a}` }, '');
```
Comment (ottenhoff) [inline rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js]: is this change purposeful?

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js [hunk 2]
```diff
@@ -10,7 +10,8 @@ class SakaiRubricAssociation extends SakaiElement {
     super();
 
     this.configurationOptions = [];
-    SakaiRubricsLanguage.loadTranslations();
+
+    SakaiRubricsLanguage.loadTranslations().then(result => this.i18nLoaded = result );
```
Comment (ottenhoff) [inline rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js]: So this definitely not going to work in IE, right?

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js (hunk 1 above)
Comment (adrianfish) [reply rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js]: I'm glad you spotted that. That is some weird paste error. Nice one!

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js (hunk 2 above)
Comment (adrianfish) [reply rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js]: We need to use webpack to build a bundle if we want it to work in IE. It's the arrow functions, mainly, as we are using a polyfill which patches in promises and fetch, stuff like that.

**Comment by adrianfish** [comment]
Those codacy errors are stupid. We need to make the jslint config a little more tolerant I'd say.

**Pull request merged**